packages/core/admin/admin/src/content-manager/hooks/useKeyboardDragAndDrop.js

```diff
@@ -17,9 +17,13 @@ import { useState } from 'react';
  *
  * @internal - You should use `useDragAndDrop` instead.
  *
- * @type {(index: number, callbacks: UseKeyboardDragAndDropCallbacks) => (event: React.KeyboardEvent<HTMLButtonElement>) => void}
+ * @type {(active: boolean, index: number, callbacks: UseKeyboardDragAndDropCallbacks) => (event: React.KeyboardEvent<HTMLButtonElement>) => void}
  */
-export const useKeyboardDragAndDrop = (index, { onCancel, onDropItem, onGrabItem, onMoveItem }) => {
+export const useKeyboardDragAndDrop = (
+  active,
```

Comment:
so is it `false` by default?
should we update the doc? 👀

Reply:
I don't think so, it's an internal hook not really supposed to be used on it's own and i'd rather have it not perform because active is Falsey (`undefined`) rather than it do run.